docs/en/rst/installing/mysql.rst

```diff
@@ -26,7 +25,8 @@ Download the MariaDB 32-bit or 64-bit MSI installer from the
 
 MariaDB has a standard Windows installer. It's ok to select a the
```

Comment:
Urk, I failed to notice this before, but we really don't need to talk about MariaDB in the MySQL docs anymore. :-) Probably just change MariaDB to MySQL everywhere in this paragraph.